cpp/doxygen/developer_guide/DEVELOPER_GUIDE.md

```diff
@@ -403,6 +403,14 @@ Functions like merge or groupby in libcudf make no guarantees about the order of
 Promising deterministic ordering is not, in general, conducive to fast parallel algorithms.
 Calling code is responsible for performing sorts after the fact if sorted outputs are needed.
 
+## libcudf does not promise consistent exception messages
```

Comment:
Maybe "specific" instead of "consistent"?
```suggestion
## libcudf does not promise specific exception messages
```
Or even:
```suggestion
## Exception messages are not API stable
```